javaspec-runner/src/main/java/info/javaspec/spec/FieldSpec.java

```diff
@@ -34,7 +34,7 @@ public void addDescriptionTo(Description suite) {
   public void run(RunNotifier notifier) {
     try {
       state = state.instantiate();
-    } catch(TestSetupFailed ex) {
```

Comment:
One of the contributing factors - the exceptions of interest would have had to be wrapped in `TestSetupFailed` for this to work.